packages/main/src/plugin/api/manifest-info.ts

```diff
@@ -27,3 +27,24 @@ export interface ManifestCreateOptions {
   // Provider to use for the manifest creation, if not, we will try to select the first one available (similar to podCreate)
   provider?: ContainerProviderConnection;
 }
+
+export interface ManifestInspectInfo {
+  engineId: string;
+  engineName: string;
+  manifests: {
+    digest: string;
+    mediaType: string;
+    platform: {
+      architecture: string;
+      features: string[];
+      os: string;
+      osFeatures: string[];
+      osVersion: string;
+      variant: string;
```

Comment:
are these params mandatory ? AFAIK some of them are optional no ? like OSFeatures

Reply:
it is just returning information, so they are not mandatory, they are just there so the API can pick it up.

Reply:
I tried with

```
curl --unix-socket /var/run/docker.sock  "http://libpod/v3.0.0/libpod/manifests/docker.io%2Fhttpd/json" | jq .
```

the fields are not there so it should be listed as optional.
                                                                                                    
```
{
  "schemaVersion": 2,
  "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
  "manifests": [
    {
      "mediaType": "application/vnd.oci.image.manifest.v1+json",
      "size": 2095,
      "digest": "sha256:6834aeaa0023c674c53a8891ce77a363579ae9a9679222f9634f95aa72c5781b",
      "platform": {
        "architecture": "amd64",
        "os": "linux"
      }
    },
    {
      "mediaType": "application/vnd.oci.image.manifest.v1+json",
      "size": 841,
      "digest": "sha256:d8081c32306039fe31141c53642d73c59678f70678e85f7506c01d15aa5a95e3",
      "platform": {
        "architecture": "unknown",
        "os": "unknown"
      }
    },
    {
      "mediaType": "application/vnd.oci.image.manifest.v1+json",
      "size": 2097,
      "digest": "sha256:623ea480a45a037db310321eb5f9b933df664701f5be02feb2164b302b2e293d",
      "platform": {
        "architecture": "arm",
        "os": "linux",
        "variant": "v5"
      }
    },
```